.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -272,10 +276,10 @@ jobs:
             cargo_dir: ~/.cargo
             cc: clang
             cxx: clang++
-          - name: macOS 12 (clang) Qt6
-            os: macos-12
+          - name: macOS 14 (clang) Qt6
+            os: macos-14
             qt_version: 6
-            aqt_version: '6.7.3'
+            aqt_version: '6.8.0'
             aqt_arch: 'clang_64'
             aqt_host: 'mac'
             dyld_framework_path: /Users/runner/work/cxx-qt/Qt/6.7.3/macos/lib
```

Comment:
the `clang_format_path` below for macOS 14 needs updating to `/Users/runner/Library/Python/3.13/bin/clang-format` as there is a different python version now